en/docs/references/define-configurable-variables.md

```diff
@@ -0,0 +1,126 @@
+# Define Configurable Variables
+
+A configurable variable allows you to define a parameter for which you specify the value when you deploy the component that includes it.
+
+You may want to define a configurable variable in the following scenarios:
+
+- A Choreo component you create is accessible to all the members of your organization. If a specific variable value contains sensitive information, you can define it as a configurable variable to avoid exposing the value in the component implementation.<br/>
+
+    For example, consider a REST API that returns a password to users whose requests meet specific conditions. In this scenario, you may not want to expose that password to everyone who can access the REST API implementation.
+
+- You may need to provide different values for a specific variable.
+
+    For example, eachncomponent user may want to provide a unique email address to receive the response.
+
+
+To understand how to define a configurable variable, let's  create a REST API that returns a configurable email address to its users  as follows:
+
+1. First, let's create the component in which you can add a configurable variable:
+
+    1. Sign in to the Choreo Console at https://console.choreo.dev.
+
+    2. Under **Create a New Component**, click **REST API** to start creating a REST API.
+   
+    3. In the **Create REST API** dialog, click **Start From Scratch**. Next, enter a unique name for the REST API and click **Create**.
+   
+        !!! Info
+            To learn how to create a REST API with more details, try the [Create Your First REST API tutorial](../tutorials/create-your-first-rest-api.md).
+
+    4. Once the **Components** page opens, click **Edit Code** to open the Web Editor.
+
+2. Now let's define the email address you want to return as a configurable by following these sub-steps:
+
+    1. Click the first **+** icon on the page, and in the **Add Constructs** pane that appears, click **Configurable**.
+   
+        ![Add Configurable](../assets/img/configurables/add-configurable.png){.cInlineImage-threeQuarters}
+   
+        The **Configurable** pane opens, displaying the template for the configurable.
```

Comment:
We are repeating this in the next step too. We might have to consider updating the next step accordingly.

Reply:
Removed repeated text.